packages/sanes-chrome-extension/src/routes/account/test/drawer.ts

```diff
@@ -13,13 +14,16 @@ const initDrawer = async (drawerComponent: React.Component): Promise<void> => {
 
   const drawerList = await findRenderedDOMComponentWithId(drawerComponent, DRAWER_HTML_ID);
   const drawerElements = (drawerList as Element).querySelectorAll('div > div:nth-of-type(2)');
-  expect(drawerElements.length).toBe(2);
+  expect(drawerElements.length).toBe(3);
 
   recoveryPhraseLink = drawerElements[0];
   expect(recoveryPhraseLink.textContent).toBe('Recovery words');
 
   requestsLink = drawerElements[1];
   expect(requestsLink.textContent).toBe('Requests');
+
+  logoutLink = drawerElements[2];
+  expect(logoutLink.textContent).toBe('Logout');
```

Comment:
I think you forgot to implement test for logout menu item click behaviour in `account/index.dom.spec.ts`.